### PR TITLE
Add final roadmap '20 review

### PR DESCRIPTION
**Pull request by lukpueh**

### CODE REVIEW

File: roadmap-reviews/2020/review_3_june_20.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+  ([#284](https://github.com/in-toto/in-toto/issues/284))
+- Major CLI documentation overhaul and automation
+  ([#341](https://github.com/in-toto/in-toto/pull/341))
+- Revise and update source code documentation and build with sphinx
+  ([#369](https://github.com/in-toto/in-toto/pull/369))
+
+
+## Working towards 1.0
+
+Despite the vast improvements in our documentation and the thus resulting
```
Comment (adityasaky) [inline roadmap-reviews/2020/review_3_june_20.md]:
```suggestion
Despite the vast improvements in our documentation and the resulting
```

Sounds a bit redundant to me

**Review by adityasaky — COMMENTED** [review]
I think this looks fine, but I'd prefer if @SantiagoTorres could sign off :smile:

File: roadmap-reviews/2020/review_3_june_20.md (hunk 1 above)
Comment (lukpueh) [reply roadmap-reviews/2020/review_3_june_20.md]: Thanks ... incorporated with e79d1d1.

File: roadmap-reviews/2020/review_3_june_20.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+  ([#364](https://github.com/in-toto/in-toto/pull/364))
+- Tests: Add tmp dir and gpg key test mixins
+  ([#345](https://github.com/in-toto/in-toto/pull/345))
+- Tests: Use constant from securesystemslib to detect GPG in tests
+  ([#352](https://github.com/in-toto/in-toto/pull/352))
+- Tests: Enhance test suite feedback on Windows
+  ([#368](https://github.com/in-toto/in-toto/pull/368))
+
+## Tangential work
+On a related side-note, together with the TUF team we have made exiting
```
Comment (SantiagoTorres) [inline roadmap-reviews/2020/review_3_june_20.md]: exciting

File: roadmap-reviews/2020/review_3_june_20.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+'21.
+
+
+### Closing remarks.
+
+We acknowledge that we missed our main roadmap '20 goal, that is the release of
+in-toto 1.0. But it should be noted that it is a close miss and we are
+confident about our decision to hold off the release until we have dealt with
+the last remaining blemish (i.e.
+[#80](https://github.com/in-toto/docs/issues/90), so that we can guarantee the
```
Comment (SantiagoTorres) [inline roadmap-reviews/2020/review_3_june_20.md]: This is 90 right?

**Review by SantiagoTorres — APPROVED** [review]
two very cosmetic things, but overall LGTM :)

File: roadmap-reviews/2020/review_3_june_20.md (hunk 2 above)
Comment (lukpueh) [reply roadmap-reviews/2020/review_3_june_20.md]: Good eyes!

File: roadmap-reviews/2020/review_3_june_20.md (hunk 3 above)
Comment (lukpueh) [reply roadmap-reviews/2020/review_3_june_20.md]: Very good eyes. It's #80. Fixed this and the typo above in force push.

**Pull request merged**